tests/snapshots/generate_plan_tests__bun.snap

```diff
@@ -0,0 +1,39 @@
+---
+source: tests/generate_plan_tests.rs
+expression: plan
+---
+{
+  "version": "0.2.9",
```

Comment:
Do we have to input the version every time for each test? I presume we have to update it on every new version as well?

Reply:
No these are never edited manually. They are used to compare the build plan output and make sure there are no changes, if there are changes you use `cargo insta review` to go through them one by one and either accept or reject the updates.

For example, after manually editing a `.snap` file and running

```
cargo insta test --review -- --test generate_plan_tests
```

I get

<img width="1054" alt="image" src="https://user-images.githubusercontent.com/3044853/180621887-4da05562-f8a7-4d56-8a69-f4554dccb74f.png">

Pressing `a` will automatically update the snapshot with the changes